api/server/sdk/volume_ops.go

```diff
@@ -491,3 +506,49 @@ func setSpecBool(current, req bool, reqSet interface{}) bool {
 	}
 	return current
 }
+
+func GetEnforcedVolSpecs(locator *api.VolumeLocator, source *api.Source, spec *api.VolumeSpec) (*api.VolumeSpec, error) {
+	if locator != nil {
+		if val, ok := locator.VolumeLabels["enforce_disable"]; ok {
```

Comment:
Nice. Add comment here so that others in the future who read this understand why this is needed.